lsp-metals.el

```diff
@@ -159,7 +159,7 @@ Should be ignored if there is no open doctor window."
             (buffer (lsp--buffer-for-file file)))
       (when buffer
         (with-current-buffer buffer
-          (seq-do #'delete-overlay lsp-metals--decorations)
+          (mapc #'delete-overlay lsp-metals--decorations)
```

Comment:
Possible improvement not related to this PR: I recently changed lsp-mode to utilize `remove-overlays`, see `lsp--remove-overlays`